packages/integration_test/lib/_callback_io.dart

```diff
@@ -60,8 +61,8 @@ class IOCallbackManager implements CallbackManager {
     // comes up in the future. For example: `WebCallbackManager.cleanup`.
   }
 
-  // Whether the Flutter surface uses an Image.
-  bool _usesFlutterImage = false;
+  // Whether the Flutter surface uses an Image (Android only).
```

Comment:
This sounds to me like using an image is Android-only, but the logic below is the reverse.

Reply:
@blasten What does `Whether the Flutter surface uses an Image` mean in this context?  I think that the surface is frozen, the screenshot taken, and then it's unfrozen on Android?

Reply:
oh - yes. Image really means that Flutter is using this class https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/embedding/android/FlutterImageView.java. 👻